BrainPortal/app/controllers/userfiles_controller.rb

```diff
@@ -1670,7 +1676,13 @@ def extract_from_archive(archive_file_name, file_type = nil, attributes = {}) #:
         :variable_text  => report
       )
     end
-
+    rescue => e
+      Message.send_message(current_user.own_group,
```

Comment:
At line 615 we have `send_message(current_user...` instead of `current_user.own_group` 

Which one is correct?